schemas/openapi-v3.1.json

```diff
@@ -0,0 +1,1340 @@
+{
+  "$id": "https://spec.openapis.org/oas/3.1/schema/2021-03-02",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "openapi": {
+      "type": "string",
+      "pattern": "^3\\.1\\.\\d+(-.+)?$"
+    },
+    "info": {
+      "$ref": "#/$defs/info"
+    },
+    "jsonSchemaDialect": {
+      "$ref": "#/$defs/uri",
+      "default": "https://spec.openapis.org/oas/3.1/dialect/base"
+    },
+    "servers": {
+      "$ref": "#/$defs/server"
+    },
```

Comment:
Schema used here is outdated pre-release variant. `servers` property declaration was fixed to be an array

https://github.com/OAI/OpenAPI-Specification/blob/1fb4f83882e135899fee4c521cc1c6161bb85125/schemas/v3.1/schema.json#L18-L23